PocketCoffea/utils/Plot.py

```diff
@@ -0,0 +1,332 @@
+import os
```

Comment:
This seems related only to the SF plots. Why it is part of the utils/Plot.py file?

Reply:
Several functions are defined in `utils/Plot.py`, not only related to the SF plots but also for the efficiency + uncertainty computation and plot of the efficiency (1D + 2D plots). These functions could be used not only for the trigger efficiency, but for any kind of efficiency that we want to compute and/or plot, therefore the idea to define a module in `utils`.
However the location of the module is not binding, we could move it in another folder if preferable.